### DISCUSSION TITLE
Where can I find the MSI installer for Windows?

### DISCUSSION THREAD

**Discussion by wra1w0** [dathere/qsv#2303] [Q&A] [answered]
Where can I find the MSI installer for Windows mentioned in README?
> For Windows, an MSI Installer wrapping the x86_64-pc-windows-msvc build is also available for download.
The release page contains only 2 `.zip` archives, neither of which contains a `.msi` installer. 
---
P.S.:  
Sorry if it's not obvious for me, but I'm not a Windows user and I'm looking for some easy way to install QSV on a Windows machine.

**Comment by jqnatividad** [comment, marked as answer]
We'll publish MSI installers again in future releases.

We skipped the past few because Windows Defender would flag qsv as malware unless we signed it.

In the interim, you can install from source or install qsv using scoop.
https://scoop.sh/#/apps?q=qsv

cc @rzmk

**Comment by datatraveller1** [comment]
As a further hint: QSV can also be used without an installer. In this case, I recommend to download the newest released windows-msvc.zip file (e.g. [.../qsv/releases/download/0.138.0/qsv-0.138.0-**x86_64-pc-windows-msvc.zip**](https://github.com/jqnatividad/qsv/releases/download/0.138.0/qsv-0.138.0-x86_64-pc-windows-msvc.zip))
Then, unpack qsv.exe to a directory from where you wish it to use. For convenient use of qsv.exe (acccess from any directory), add a path to this directory (for an instruction see e.g. https://www.computerhope.com/issues/ch000549.htm).
qsv.exe is a native Windows exe file without any additional software needed.
For using qsv.exe, you'll need to access powershell or the command prompt (cmd.exe).
To access the command prompt using the Run window, press the Win + R keys on your keyboard. Then, type cmd and press Enter or click/tap OK.

**Comment by rzmk** [comment]
@wra1w0 There is an MSI installer now available for qsv v1.0.0: https://github.com/dathere/qsv/releases/latest
